utils/symbolicate-linux-fatal.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+
+# TODOs:
+# * verbose output
+# * search symbols by name for the not <unavailable> ones
+
+import sys
+import lldb
+from subprocess import Popen, PIPE
+
+dynlibs = {}
+def process_ldd(lddoutput):
+    global dynlibs
```

Comment:
This line, `global dynlibs` is not needed. Since you define `dynlibs` in a global scope above, you are able to reference it throughout this file. This is why you're able to reference it in the `process_stack` function below.

Also, I believe the Python linter will raise this issue, but I think that typically Python globals tend to be in `UPPER_SNAKE_CASE`.